src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -204,6 +204,27 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('publish_microsoft_endpoints', publish_microsoft_endpoints_type)
         c.argument('publish_internet_endpoints', publish_internet_endpoints_type)
 
+    with self.argument_context('storage account private-endpoint-connection',
+                               resource_type=ResourceType.MGMT_STORAGE) as c:
+        c.argument('private_endpoint_connection_name', options_list=['--name', '-n'],
+                   help='The name of the private endpoint connection associated with the Storage Account.')
+    for item in ['approve', 'reject', 'show', 'delete']:
+        with self.argument_context('storage account private-endpoint-connection {}'.format(item),
+                                   resource_type=ResourceType.MGMT_STORAGE) as c:
+            c.argument('private_endpoint_connection_name', options_list=['--name', '-n'], required=False,
```

Comment:
>required=False [](start = 90, length = 14)

this is a question, what is the default value for required? from your sample, it seems we can use connection_id or connection_name + account_name +rpName, why are the later parameters marked as required=false

Reply:
`required` means it has to be provided when using the command.
As you know, we have two options to manage the connection. With the two options, connection_id can be none if you use option2, and connection_name+account_name+rgName also can be none if you use option1.
In this way, no one is always required for the command.

Reply:
why connection_id does not have required=false marked? I suppose required is false by default?